libraries/api-documenter/src/index.ts

```diff
@@ -18,7 +18,7 @@ console.log(os.EOL + colors.bold(`api-documenter ${myPackageJson.version}` + os.
 
 const docItemSet: DocItemSet = new DocItemSet();
 
-const dataFolder: string = path.join(__dirname, '../files');
+const dataFolder: string = path.join(__dirname, '../local');
```

Comment:
>local [](start = 52, length = 5)

Is this folder only used by api-doc or for general usage?

Reply:
General usage.  I'm planning to update the sp-client .gitignore as well, and document our standardized folder names on the wiki.

Reply:
General usage.  I'm planning to update the sp-client .gitignore as well, and document our standardized folder names on the wiki.

---
In reply to: [144613385](https://github.com/Microsoft/web-build-tools/pull/393#discussion_r144613385) [](ancestors = 144613385)